### ISSUE TITLE
Add `Cross-Origin-Resource-Policy: cross-origin` header to CDN asset to support sites with COEPs

### ISSUE THREAD

**Issue by fabianuribe** [vercel/analytics#39]
Currently the Analytics debug script (`script.debug.js`) gets blocked on development environment when the site has a Cross-Origin Embedder Policy with a `require-corp` value. (e.g. Required for enabling APIs that require site isolation such as [SharedArrayBuffer](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer))

```
GET https://cdn.vercel-insights.com/v1/script.debug.js 
net::ERR_BLOCKED_BY_RESPONSE.NotSameOriginAfterDefaultedToSameOriginByCoep 200
```

The fix usually involves simply adding `Cross-Origin-Resource-Policy: cross-origin` header to the resource served from the CDN.

More information about COEPs, COOPs and CORPs: https://web.dev/coop-coep/

**Note**: Most likely low priority, given the script is proxied on production environment and hence served from the same origin.

https://github.com/vercel/analytics/blob/554bb2c6e0ee6e30cf6f576ccd820cbcc8a37af3/packages/web/src/generic.ts#L20-L23

**Comment by tobiaslins** [comment]
Hey @fabianuribe ,

Thanks for the heads up, the header is now set for the script :)
